components/dictionary.txt

```diff
@@ -184,9 +184,12 @@ lv
 LV
 LVs
 lysis
+MacDown
+MacOS
 macrophage
 Malhotra
 McInnes
+md
 medulloblastoma
```

Comment:
Minor, but I think we can revert these dictionary changes. 

Slightly more significant (but hard for me to comment directly on): Can you add the **Cheatsheets** heading to the TOC in CONTRIBUTING.md? If you have [doctoc](https://github.com/thlorenz/doctoc) installed, that is the "usual" way, but I use a [plugin for vscode](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) that does it automatically too, and you can always just add the link manually.
```suggestion
macrophage
Malhotra
McInnes
medulloblastoma
```